signinapp/dbadmin.py

```diff
@@ -47,14 +47,14 @@ def init_app(app):
     )
 
     flask_admin.add_views(
-        AuthModelView(Badge, db.session, endpoint="badge"),
-        AuthModelView(Event, db.session, endpoint="adminevent"),
+        AuthModelView(Badge, db.session, endpoint="admin_badge"),
+        AuthModelView(Event, db.session, endpoint="admin_event"),
         AuthModelView(EventType, db.session, endpoint="eventtype"),
         AuthModelView(Guardian, db.session, endpoint="guardian"),
         AuthModelView(Role, db.session, endpoint="role"),
         AuthModelView(Student, db.session, endpoint="student"),
         AuthModelView(Subteam, db.session, endpoint="subteam"),
-        AuthModelView(User, db.session, endpoint="adminuser"),
+        AuthModelView(User, db.session, endpoint="admin_user"),
         AuthModelView(Stamps, db.session, endpoint="stamps"),
```

Comment:
Aren't all of these at /dbadmin/* do they need to be re-named?

Reply:
And if they aren't, then I think we should just update all of these with the `admin_` prefix.

Reply:
These are the endpoint, which is different from the url prefix. I'm updating the endpoint names though.